src/GraphQL.Tests/Execution/AstFromValueTests.cs

```diff
@@ -8,11 +8,17 @@ namespace GraphQL.Tests.Execution
 {
     public class AstFromValueTests
     {
+        [Fact]
+        public void throws_for_null_graphtype()
+        {
+            Should.Throw<ArgumentNullException>(() => ((IGraphType)null).ToAST(null));
```

Comment:
```suggestion
            Should.Throw<ArgumentNullException>(() => ((IGraphType)null).ToAST(true));
```
I think it's better to pass non-null argument so it's clear that exception belongs to `this` argument.